lib/liquid/standardfilters.rb

```diff
@@ -918,6 +948,50 @@ def sum(input, property = nil)
 
     attr_reader :context
 
+    def filter_array(input, property, target_value, &block)
+      ary = InputIterator.new(input, context)
+
+      return [] if ary.empty?
+
+      block.call(ary) do |item|
+        if target_value.nil?
+          fetch_property(item, property)
+        else
+          fetch_property(item, property) == target_value
+        end
+      rescue TypeError
+        raise_property_error(property)
+      rescue NoMethodError
+        return nil unless item.respond_to?(:[])
+        raise
+      end
+    end
+
+    def fetch_property(drop, property_or_keys)
```

Comment:
```suggestion
    def get_property(drop, property_or_keys)
```

I would probably name this method `get_`. My understanding of the wording, is that get is internal, and fetch is external.

[Other languages](https://laravel.com/docs/11.x/helpers) also use `get` for this kind of thing.